### PR TITLE
Add FR parsing rules

### PR DESCRIPTION
**Pull request by skyman503**
This PR introduces parsing rules for the French data model.

### CODE REVIEW

File: model/countries/FR/FR-parsing-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  output:
+    street-address-alternative-1: "1661 Place Charles de Gaulle"
+    building-location: "1661 Place Charles de Gaulle"
+    street: "Place Charles de Gaulle"
+    building: "1661"
+- id: "Test 5"
+  type: street-address-alternative-1
+  input: "Appartament 36\n1661 Place Charles de Gaulle"
+  output:
+    street-address-alternative-1: "Appartament 36\n1661 Place Charles de Gaulle"
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:89]: In french, the correct spelling is `Appartement`. Let's correct that on all the examples.

File: model/countries/FR/FR-parsing-rules.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    street-address-alternative-1: "Appartament 36, 1661 Place Charles de Gaulle"
+    building-location: "1661 Place Charles de Gaulle"
+    street: "Place Charles de Gaulle"
+    building: "1661"
+    address-overflow: "Appartament 36"
+- id: "Test 8"
+  type: street-address-alternative-1
+  input: "Appartament 1, 58 rue du Gue Jacquet"
+  output:
+    street-address-alternative-1: "Appartament 1, 58 rue du Gue Jacquet"
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:116]: Can we test multiple ways to spell the Appartement (e.g. Appt, Apt., etc)?

File: model/countries/FR/FR-parsing-rules.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    building-location: "58 rue du Gue Jacquet"
+    street: "rue du Gue Jacquet"
+    building: "58"
+    address-overflow: "Appartament 1"
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:120]: Can we add tests for when the street address contains things like Battiment or building name
- Apt. 45, Résidence Les Lilas, 12 Rue de la Paix
- Bâtiment A, 5 Avenue des Champs-Élysées, 75008 Paris, France

File: model/countries/FR/FR-parsing-rules.yaml (hunk 1 above)
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml]: Ahh sorry for that, I check that but probably copied the value from the wrong side of Google translate 😬

File: model/countries/FR/FR-parsing-rules.yaml (hunk 2 above)
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml]: Done.

**Review by norgevz — APPROVED** [review]
LGTM

I will follow-up this PR with other improvements that we've discussed.

**Pull request merged**